resolwe_bio/processes/workflows/trim-align-quant.yml

```diff
@@ -0,0 +1,167 @@
+- slug: workflow-trim-align-quant
+  name: Trim, align and quantify using a library as a reference.
```

Comment:
This name lacks the shRNA reference. Think about if `shRNA primary analysis` or something similar would be better.